index.html

```diff
@@ -0,0 +1,324 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="images\logo.png" type="image/x-icon">
+    <link rel="stylesheet" href="http://bloom-component-library.vercel.app/components\component.css">
```

Comment:
Instead of the backslash, there should be a forward slash. `http://bloom-component-library.vercel.app/components/component.css`